packages/dds/sequence/src/test/fuzz/fuzzUtils.ts

```diff
@@ -57,6 +58,14 @@ export function isRevertibleSharedString(s: ISharedString): s is RevertibleShare
 	return (s as RevertibleSharedString).revertibles !== undefined;
 }
 
+export interface OracleSharedString extends ISharedString {
+	oracle: SharedStringOracle;
+}
+
+export function hasOracle(s: ISharedString): s is OracleSharedString {
+	return (s as OracleSharedString).oracle !== undefined;
```

Comment:
we can make this a bit safer. 
```suggestion
	return "oracle" in s && s.oracle instanceof SharedStringOracle;
```